Exception/CmpException/MissingAccountException.php

```diff
@@ -29,16 +29,29 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  * @author Nosto Solutions Ltd <contact@nosto.com>
- * @copyright 2020 Nosto Solutions Ltd
+ * @copyright 2021 Nosto Solutions Ltd
  * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
  *
  */
 
-namespace Nosto\Cmp\Exception;
+namespace Nosto\Cmp\Exception\CmpException;
 
-use Nosto\NostoException;
+use Exception;
+use Throwable;
 
-class MissingCookieException extends NostoException
+class MissingAccountException extends Exception
```

Comment:
Make sure all exception classes follow the same inheritance. If there is no need for any extra common setup, extend Exception and remove CmpException class